### PR TITLE
Add acronym generator

### PR DESCRIPTION
**Pull request by bmulvihill**
Use x-common to generate Acronym tests

### CODE REVIEW

File: lib/acronym_cases.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class AcronymCase < OpenStruct
+  def name
+    'test_%s' % expected.downcase
```
Comment (Insti) [inline lib/acronym_cases.rb]: `description` in the `canonical-data.json` file provides information about what the test is testing for.
Either: use the description as the test name.
add the description as a comment in the test cases.

**Comment by Insti** [comment]
:tada: nice work! I have a few minor issues with the generator but everything else looks really good. Thanks!

**Comment by bmulvihill** [comment]
@Insti I made requested changes

**Comment by Insti** [comment]
Thanks a lot @bmulvihill! :heart:

**Pull request merged**